modules/instrument_builder/js/instrument_builder.instrument.js

```diff
@@ -120,10 +120,15 @@ var Instrument = {
 
                         // Add dropdown and special naming when no date format is set
                         // (i.e when addDateElement() is used)
-                        if (element.Options.dateFormat === "") {
-                            elName = elName + "_date";
```

Comment:
i'm not sure if we should be removing this line?

Reply:
@zaliqarosli thanks! Unsure why I removed it. Sorry!

Reply:
Hi @zaliqarosli I did have to add this code back:
```
if (!elName.includes('_date')) {
   // prevents double _date if load & save linst file.
  elName = elName + "_date";
}
```
It turns out that I had it origionally because the load of a previous instrument file will add `_date` again to the elName and even if already existing to the name.

See test1.linst without the above code:
```
table{@}test1
title{@}test1
date{@}Date_taken{@}Date of Administration{@}2006{@}2012
static{@}Candidate_Age{@}Candidate Age (Months)
static{@}Window_Difference{@}Window Difference (+/- Days)
select{@}Examiner{@}Examiner{@}NULL=>''
header{@}{@}test1
date{@}test1date_date{@}test1date{@}2020{@}2021{@}Date
select{@}test1date_date_status{@}{@}NULL=>''{-}'not_answered'=>'Not Answered'

```
and test2.linst loaded from test1.linst:
```
table{@}test2
title{@}test2
date{@}Date_taken{@}Date of Administration{@}2006{@}2012
static{@}Candidate_Age{@}Candidate Age (Months)
static{@}Window_Difference{@}Window Difference (+/- Days)
select{@}Examiner{@}Examiner{@}NULL=>''
header{@}{@}test1
date{@}test1date_date_date{@}test1date{@}2020{@}2021{@}Date
select{@}test1date_date_date_status{@}{@}NULL=>''{-}'not_answered'=>'Not Answered'

```